### PR TITLE
link searchjob feature to homepage and show the data from models

### LINKED ISSUE #38: Connect search results URL to homepage

### PR DESCRIPTION
**Pull request by itaiv95**
made the following:
1.link the "find it" on home-page to searchjob html page (through url and view files)
2.create search job html page that show database models data
3. show the relevat data by date (view file)

Fixes: #38

### CODE REVIEW

File: searchJob/urls.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+from . import views
+
+app_name = 'searchJob'
+
+urlpatterns = [
+    url(r'^$', views.jobs),
```
Comment (guy9050) [inline searchJob/urls.py:7]: did you check this? I think you're missing an attribute  url(r'^$', views.jobs, name=...)

Comment (itaiv95) [reply searchJob/urls.py:7]: yes,  i checked that and it work well. 
i saw django documentation and i will fix it right now 
thank you

Comment (itaiv95) [reply searchJob/urls.py:7]: yes, i check that and it work well.

**Review by bardielle — APPROVED** [review]
/lgtm

File: joboard/urls.py [hunk 2]
```diff
@@ -14,4 +14,5 @@
     path('signup/', user_views.signup_view, name='signup'),
     path('login/', auth_views.LoginView.as_view(template_name='signup/login.html'), name='login'),
     path('logout/', auth_views.LogoutView),
+    url(r'^jobs/', include('searchJob.urls')),
```
Comment (Yarboa) [inline joboard/urls.py:17]: sorry i lost you,
Why do you have two urls?
joboard/urls
searchjob/urls
Is it 2 applications?

Comment (itaiv95) [reply joboard/urls.py:17]: > sorry i lost you,
> Why do you have two urls?
> joboard/urls
> searchjob/urls
> Is it 2 applications?

Yes, we have 3 apps that every one of them on separate directory and work as an app

**Pull request merged**